src/Server.ts

```diff
@@ -1,28 +1,49 @@
 import * as express from 'express';
 import Iconfig from './config/Iconfig';
+import * as bodyParser from 'body-parser';
+import { errorHandler, notFoundRoute } from './libs/routes/index';
```

Comment:
no need to write index while importing. Remove it

Reply:
done